File: spring-boot-modelmapper-dto-example-demo/src/main/java/_003_javabeans/valueobject/Company.java
package _003_javabeans.valueobject;

/**
*
@author Huseyin_Aydin
@since 1994
@category Spring Boot Examples
*
**/

public class Company {	
	
	private int companyId; //Sınıf değişkenleri var.
	private String name;
	
	public int getCompanyId() {
		return companyId;
	}
	public void setCompanyId(int companyId) {
		this.companyId = companyId;
	}
	public String getName() {
		return name;
	}
	public void setName(String name) {
		this.name = name;
	}
}

File: spring-kotlin-crud-example-demo/spring-kotlin-crud-master/src/main/kotlin/tr/com/huseyinaydin/spring/kotlin/crud/api/model/Company.kt
package tr.com.huseyinaydin.spring.kotlin.crud.api.model

import javax.persistence.Entity
import javax.persistence.Id
import javax.persistence.Table
import javax.persistence.GeneratedValue

/**
 *
@author Huseyin_Aydin
@since 1994
@category Spring Boot Examples
 *
 **/

@Entity
@Table
data class Company(
		@Id
		@GeneratedValue
		val id: Int = 0,
		val name: String = "",
		val address: String = "")